catppuccin_tmux.conf

```
@@ -0,0 +1,105 @@
+
+
+  # Batch options for `./catppuccin.tmux`
+  # add_tmux_batch_option "@catppuccin_status_default"
+  # add_tmux_batch_option "@catppuccin_status_justify"
+  # add_tmux_batch_option "@catppuccin_status_background"
+  # add_tmux_batch_option "@catppuccin_menu_style"
+  # add_tmux_batch_option "@catppuccin_menu_selected_style"
+  # add_tmux_batch_option "@catppuccin_menu_border_style"
+  # add_tmux_batch_option "@catppuccin_pane_status_enabled"
+  # add_tmux_batch_option "@catppuccin_pane_border_status"
+  # add_tmux_batch_option "@catppuccin_pane_border_style"
+  # add_tmux_batch_option "@catppuccin_pane_active_border_style"
+  # add_tmux_batch_option "@catppuccin_pane_left_separator"
+  # add_tmux_batch_option "@catppuccin_pane_middle_separator"
+  # add_tmux_batch_option "@catppuccin_pane_right_separator"
+  # add_tmux_batch_option "@catppuccin_pane_number_position"
+  # add_tmux_batch_option "@catppuccin_window_separator"
+  # add_tmux_batch_option "@catppuccin_window_left_separator"
+  # add_tmux_batch_option "@catppuccin_window_right_separator"
+  # add_tmux_batch_option "@catppuccin_window_middle_separator"
+  # add_tmux_batch_option "@catppuccin_window_number_position"
+  # add_tmux_batch_option "@catppuccin_window_status"
+  # add_tmux_batch_option "@catppuccin_status_left_separator"
+  # add_tmux_batch_option "@catppuccin_status_right_separator"
+  # add_tmux_batch_option "@catppuccin_status_connect_separator"
+  # add_tmux_batch_option "@catppuccin_status_fill"
+  # add_tmux_batch_option "@catppuccin_status_modules_left"
+  # add_tmux_batch_option "@catppuccin_status_modules_right"
+
+  # run_tmux_batch_commands
+
+  # status general
+  # local status_default status_justify status_background message_background
+  # status_default=$(get_tmux_batch_option "@catppuccin_status_default" "on")
+  # shellcheck disable=SC2121
+
+
+set status-left-length "100"
+set status-right-length "100"
+
+
+%if "#{==:#{@catppuccin_status_background},theme}"
+  set status-bg "#{@thm_bg}"
+  set -gF @message_background "#{@thm_gray}"
+%elif "#{==:#{@catppuccin_status_background},default}"
+  set status-bg "default"
+  set -g @message_background "default"
+%else
+  # Treat @catppuccin_status_background as a format string.
+  set -gF @message_background "#{E:@catppuccin_status_background}"
+  set -F status-bg "#{@message_background}"
+%endif
+
+# messages
+set -gF message-style "fg=#{@thm_cyan},bg=#{@message_background},align=centre"
+set -gF message-command-style "fg=#{@thm_cyan},bg=#{@message_background},align=centre"
+
+# menu
+set -F menu-style "#{E:@catppuccin_menu_style}"
+set -F menu-selected-style "#{E:@catppuccin_menu_selected_style}"
+set -F menu-border-style "#{E:@catppuccin_menu_border_style}"
+
+
+# Source the status line modules
+
+# panes
+pane_format=$(load_modules "pane_default_format" "$modules_custom_path" "$modules_pane_path")
+
+set -wF pane-active-border-style "#{E:@catppuccin_active_border_style}"
```

```suggestion
set -wF pane-active-border-style "#{E:@catppuccin_pane_active_border_style}"
```
(also -g instead of -w)